sdk/v2/data.go

```diff
@@ -183,9 +182,9 @@ func (integration *Integration) Clear() {
 	integration.Data = []*EntityData{} // empty array preferred instead of null on marshaling.
 }
 
-// toJSON returns the integration as a JSON string. If the pretty attribute is
-// set to true, the JSON will be idented for easy reading.
-func (integration *Integration) toJSON(pretty bool) (string, error) {
+// toJSON serializes integration as JSON. If the pretty attribute is
+// set to true, the JSON will be indented for easy reading.
+func (integration *Integration) toJSON(pretty bool) ([]byte, error) {
```

Comment:
In a next iteration we could call this method `MarshalJSON` following the JSON Marshaler Interface from stdlib https://golang.org/pkg/encoding/json/#Marshaler